enhancements/update/upgrades-blocking-on-ack.md

```diff
@@ -0,0 +1,211 @@
+---
+title: upgrade-acknowledgment-gate
+authors:
+  - "@jottofar"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@bparees"
+  - "@sdodson"
+  - "@csrwng"
+  - "@relyt0925"
+  - "@jeremyeder"
+approvers:
+  - TBD 
```

Comment:
We should get someone to sign up to be approvers for when we have consensus. Maybe @sdodson is a good person?